### DISCUSSION TITLE
What is the Battery / "Energy Impact"?

### DISCUSSION THREAD

**Discussion by olso** [raphamorim/rio#48] [Q&A] [answered]
Does this project suffer the same issues as hyper terminal?

**Comment by raphamorim** [comment]
Hey @olso thanks for the discussion, could elaborate about the issues? You are talking about memory consumption?

**Reply by olso** [reply]
<img width="969" alt="Screenshot 2023-05-25 at 15 09 32" src="https://github.com/raphamorim/rio/assets/7975158/283f211c-e0a3-46c9-adc2-f5cb9ba56bd4">

this energy impact on laptop battery life

**Reply by raphamorim** [reply]
I see, curious about the comparison of Rio and Hyper usage. I believe should be way less than Hyper in terms of memory usage due to the nature of how Rio was written, doesn't use a browser behind for example.

**Comment by drewbitt** [comment, marked as answer]
<img width="1373" alt="image" src="https://github.com/raphamorim/rio/assets/26885897/388e3296-7b2c-41f9-a3d0-4afdd2f8827f">

While both were running `sl`. Rio was overall 2-3 below iTerm for energy impact, although also once higher. 
Both sit at 0 when not in use.
